src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -161,6 +179,31 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
             </a>
           </li>
         ) : null}
+        {location ? (
+          <li>
+            <p css={[BylineCss.location]}>
+              <span
+                css={[
+                  BylineCss.reportingFromText,
+                  getSansRegularItalic(service),
+                  getBrevier(script),
+                ]}
+              >
+                {reportingFromTranslated}
+              </span>
+              <br />
```

Comment:
Can we do this with CSS please?

Reply:
Yep, I've removed the br tag and applied block styling to the `location`